app/code/UpdateDocsCronTask.php

```diff
@@ -3,11 +3,16 @@
 class UpdateDocsCronTask implements CronTask 
 {
     /**
+     * @var SS_HTTPRequest $request The request object the controller was called with.
+    */
+    protected $request = null;
+
+    /**
      * @return string
      */
     public function getSchedule() 
     {
-	return "0 20 * * *";
+	return "0 22 * * *";
```

Comment:
Too fast! I changed it to 22 temporarily to test locally since it is 22 where I am :)